zebrad/src/components/mempool/storage/verified_set.rs

```diff
@@ -225,5 +265,6 @@ impl VerifiedSet {
             "zcash.mempool.size.bytes",
             self.transactions_serialized_size as _
         );
+        metrics::gauge!("zcash.mempool.total_cost.bytes", self.total_cost as _);
```

Comment:
Note: `total_` has a special meaning to Prometheus.

> `total` (for a unit-less accumulating count)

https://prometheus.io/docs/practices/naming/

```suggestion
        metrics::gauge!("zcash.mempool.cost.bytes", self.total_cost as _);
```